### PR TITLE
more info

### PR DESCRIPTION
**Pull request by alexlee-hs**

### CODE REVIEW

File: PADx_scraper.py [hunk 1]
```diff
@@ -75,12 +114,12 @@ def main():
...
-    rem_soup = BeautifulSoup(rem_plain_txt, features="html.parser")
-    print("Scrapping...")
-
-    scrape(rem_soup, file)
+    
+    print("Scraping...")
+    start = time.perf_counter()
+    scrape(url, file)
+    end = time.perf_counter()
+    print(f"Scraping completed in {end - start} seconds.")
```
Comment (AndrooTheChen) [inline PADx_scraper.py]: Love the duration timer! One nitpick: print only 2 or 3 decimal places, the rest are negligible and waste screen space

File: PADx_scraper.py [hunk 2]
```diff
@@ -18,45 +21,81 @@ def remove_non_ascii_1(text):
...
+
+        f_rem_urls.write(f"{mon_img}@ {mon_name}@ {mon_att}@ {mon_rarity}@ {mon_types}@ {mon_series}\n") 
+
+        time.sleep(0.1)
+        printProgressBar(i + 1, numlinks, prefix = 'Progress: ', suffix = 'Complete', length = 50)
+        
     f_rem_urls.close()
     print("Finished!")
 
+def printProgressBar (iteration, total, prefix = '', suffix = '', decimals = 1, length = 100, fill = '█', printEnd = "\r"):
```
Comment (AndrooTheChen) [inline PADx_scraper.py:73]: Love this

File: PADx_scraper.py [hunk 3]
```diff
@@ -75,12 +114,12 @@ def main():
...
-    scrape(rem_soup, file)
+    
+    print("Scraping...")
+    start = time.perf_counter()
+    scrape(url, file)
+    end = time.perf_counter()
+    print(f"Scraping completed in {end - start} seconds.")
 
 if __name__ == "__main__":
     main()
```
Comment (AndrooTheChen) [inline PADx_scraper.py:108]: Mind adding a newline at the end? my bad I didn't add one earlier.

File: PADx_scraper.py [hunk 4]
```diff
@@ -18,45 +21,81 @@ def remove_non_ascii_1(text):
...
     return ''.join(i for i in text if ord(i)<128)
 
-def scrape(rem_soup, file):
+def scrape(url, file):
     """
     Take in the formatted HTML, scrape for the desired elements,
     and write to a file generating all the desired URLS
     """
     # Open file to write output to
     # f_rem_urls = open("rem_urls.txt", "w")
```
Comment (AndrooTheChen) [inline PADx_scraper.py:23]: Feel free to please get rid of this line here.

**Review by AndrooTheChen — CHANGES_REQUESTED** [review]
First of all amazing! Far exceeds my expectations. Just a few nitpicks and we can merge this. Overall great work 👍

File: PADx_scraper.py (hunk 1 above)
Comment (alexlee-hs) [reply PADx_scraper.py]: Yup. Good point.

File: PADx_scraper.py (hunk 2 above)
Comment (alexlee-hs) [reply PADx_scraper.py:73]: Thanks!

File: PADx_scraper.py (hunk 3 above)
Comment (alexlee-hs) [reply PADx_scraper.py:108]: Took 2 seconds ;)

File: PADx_scraper.py [hunk 5]
```diff
@@ -66,28 +60,17 @@ def scrape(url, file):
...
-
+        count += 1
         time.sleep(0.1)
         printProgressBar(i + 1, numlinks, prefix = 'Progress: ', suffix = 'Complete', length = 50)
-        
+    
+    printProgressBar(numlinks, numlinks, prefix = 'Progress: ', suffix = 'Complete', length = 50)
     f_rem_urls.close()
-    print("Finished!")
+    print(f"Finished! {count} monsters successfully scraped.")
```
Comment (AndrooTheChen) [inline PADx_scraper.py:71]: Nice use of f-strings

File: PADx_scraper.py [hunk 6]
```diff
@@ -119,7 +102,7 @@ def main():
     start = time.perf_counter()
     scrape(url, file)
     end = time.perf_counter()
-    print(f"Scraping completed in {end - start} seconds.")
+    print("Scraping completed in {:.2f} seconds.\n" .format(end - start))
```
Comment (AndrooTheChen) [inline PADx_scraper.py:105]: much nicer to read heh

**Review by AndrooTheChen — APPROVED** [review]
LGTM! Thanks and great work!

**Pull request merged**